src/ripple/protocol/impl/STAmount.cpp

```diff
@@ -1261,6 +1261,8 @@ divRound (STAmount const& num, STAmount const& den,
 std::uint64_t
 mulDiv(std::uint64_t value, std::uint64_t mul, std::uint64_t div)
 {
+    if ((value == 0 || mul == 0) && div != 0)
+        return 0;
```

Comment:
@vinniefalco `mulDiv` has 3 callers in this branch:

1. `LoadFeeTrack::scaleFeeBase`
  * Only non-test one caller in `ripple::RPC::checkFee`. All parameters guaranteed to be non-zero. https://github.com/ximinez/rippled/blob/fee-multi/src/ripple/rpc/impl/TransactionSign.cpp#L647-L655
2. `LoadFeeTrack::getJson`
  * Not actually used.
3. `mulDivNoThrow`
  * Used exclusively by `TxQ`, which is expecting this behavior.

So this absolutely can not affect transaction processing. Now if I had made the change in `lowestTerms`, we'd have something to talk about. (It will have a divide by 0 exception if either parameter is 0.)